examples/call-v-from-python/README.md

```diff
@@ -0,0 +1,5 @@
+A simple example to show how to call function written in v from python
```

Comment:
Please rename the folder to `examples/call_v_from_python`. In general, dashes can create more problems compared to underscores, due to command line option parsing.

Reply:
Done

Reply:
"... how to call **a** function..." just to be grammatical.

Reply:
Updated